src/signalcorr.jl

```diff
@@ -56,6 +56,8 @@ before computing the autocovariance.
 If `x` is a vector, `r` must be a vector of the same length as `x`.
 If `x` is a matrix, `r` must be a matrix of size `(length(lags), size(x,2))`, and
 where each column in the result will correspond to a column in `x`.
+
+The output is normalized by `size(x, 1)`. See `autocor!` for normalization including the variance.
```

Comment:
To be honest I did found it a bit meaningless to talk about normalizing with the size. I will remove it and simply say that this version does not normalize with the variance., etc. etc.